src/utils.jl

```diff
@@ -1,9 +1,8 @@
 """
+sample!([rng,] a, k; exclude = ())
+
```

Comment:
Why are we allowing what appears to be a vector of random number generators here?